src/Emoticon.js

```diff
@@ -0,0 +1,18 @@
+import React, { useState } from 'react';
+
+function Emoticon(props) {
```

Comment:
The attempt is good, but here you can control only the state of this component

But we need a parent component state because logic around of array of icons in the parent component